prime-router/src/main/kotlin/azure/WorkflowEngine.kt

```diff
@@ -461,7 +461,7 @@ class WorkflowEngine(
                 val time = receiver.timing.nextTime()
                 // Always force a batched report to be saved in our INTERNAL format
                 val batchReport = report.copy(bodyFormat = Report.Format.INTERNAL)
-                val event = ReceiverEvent(Event.EventAction.BATCH, receiver.fullName, time)
+                val event = BatchEvent(Event.EventAction.BATCH, receiver.fullName, time)
```

Comment:
Can / should this be removed? The `decider` should be the only thing triggering batch events.

Reply:
Unfortunately the dispatchReport function uses some information from the event for the blob upload and task insert - it does not actually put it on the queue.  This should be refactored in a future release.

Reply:
ActionHistory line 377 is the reason this even would not get sent to the queue